CorsixTH/Lua/dialogs/resizables/directory_browser.lua

```diff
@@ -70,6 +71,26 @@ function InstallDirTreeNode:createNewNode(path)
   return InstallDirTreeNode(path)
 end
 
+local function is_iso(name)
+  if name == nil then
+    return false
+  end
+
+  local ext = 'iso'
+  if string.sub(name:lower(), -string.len(ext)) == ext then
+    return true
+  end
```

Comment:
Shouldn't it always return a boolean? ie
``` lua
return string.sub(name:lower(), -string.len(ext)) == ext
```

Reply:
:+1: